get-started/composer-collections.md

```diff
@@ -6,9 +6,11 @@ publish: true
 position: 90
 ---
 
-## Organize Requests into Collections
+# Organize Requests into Collections
 
-With the Fiddler Everywhere client, you can save and organize multiple composed requests via the **Requests** list. Collections of requests can be arranged in different folders from where they could be reloaded and executed promptly. The **Requests** list is located to the left (below the **Sessions** list) and can hold multiple collections.
+With the Fiddler Everywhere client, you can save and organize multiple composed requests via the [**Requests**](https://docs.telerik.com/fiddler-everywhere/user-guide/requests-list) list. Collections of requests can be arranged in different folders from where they could be reloaded and executed promptly. The **Requests** list is located to the left (below the **Sessions** list) and can hold multiple collections.
```

Comment:
The link in Requests can be done with a **slug** instead of direct link. This will ensure that the link remains operation even if we move the articles around. I think the correct slug here is **saved-requests-tab** - the link should look like this:
[**Requests**]({%slug saved-requests-tab%})